src/beamlime/applications/daemons.py

```diff
@@ -61,3 +74,36 @@ async def run(self) -> None:
         )
 
         self.info("Data streaming finished...")
+
+
+class FakeListener(DaemonInterface):
+    messenger: MessageRouter
+
+    def __init__(self, nexus_structure: dict):
+        self._group = json_nexus_group(nexus_structure)
+
+    @classmethod
+    def from_file(cls, path: Path):
+        '''Read nexus structure from json file'''
+        with open(path) as f:
+            nexus_structure = json.load(f)
+        return cls(nexus_structure)
+
+    async def run(self) -> None:
+        self.info("Fake data streaming started...")
+
+        await self.messenger.send_message_async(
+            RunStart(
```

Comment:
`RunStart` is a bit confusing.
I thought `StatelessWorkflow` won't know about run-start and `KafkaListener` will handle them itself.
For now, it can only send an empty structure and it will be able to fill some data into the structure  as #153 implemented.

Reply:
I would just ignore this for now, and fix in the follow-up when we add events. Otherwise this is ill-defined and a waste of time to "fix" here.